app/Http/Controllers/InvoiceController.php

```diff
@@ -9,8 +9,16 @@ class InvoiceController extends Controller
 {
     public function index()
     {
-        $invoices = DB::table('invoices')->get();
-        // dd($invoices);
+        $invoices = DB::table('invoices')
+            ->join('customers', 'invoices.customer_id', '=', 'customers.id')
+            ->get([
+                'invoices.id AS id',
```

Comment:
Without this, the `id` ends up being `customers.id` from the table join because both tables have a column called `id`.